zerver/webhooks/slack/doc.md

```diff
@@ -1,34 +1,79 @@
-Get Zulip notifications from Slack for messages on your team's
-public channels!
+# Zulip Slack integration
 
-See also the [Slack-compatible webhook](/integrations/doc/slack_incoming).
+Get Zulip notifications for messages on your team's public Slack
+channels! You can choose to map each **Slack channel** either to a
+**Zulip topic** or to a **Zulip channel**.
 
-1. {!create-channel.md!}
+See also the [Zulip Slack incoming webhook integration][1].
+
+!!! warn ""
+
+    Using [Slack's legacy Outgoing Webhook service][5] is no longer
+    recommended. Follow these instructions to switch to the new
+    [Slack Events API][3].
+
+{start_tabs}
+
+1. To map each Slack channel to a Zulip topic, [create one channel][2]
+   you'd like to use for Slack notifications. Otherwise, for each public
+   Slack channel, [create a Zulip channel][2] with the same name.
```

Comment:
Do we support having a non-identity name mapping? Might be important for a channel in a Slack Connect setup where the channel in the third-party org is not what you want in your Zulip org.

Reply:
Added this TODO to #30827 for tracking.

Reply:
hmm, I don't think we have support for that.